templates/layouts/mojolicious.html.ep

```diff
@@ -8,8 +8,14 @@
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <title><%= title || 'Mojolicious - Perl real-time web framework' %></title>
     %= javascript '/mojolicious/jquery/jquery.js'
-    %= javascript '/mojolicious/prettify/run_prettify.js'
-    %= stylesheet '/mojolicious/prettify-mojo-light.css'
+    %= javascript '/mojolicious/highlight.js/highlight.min.js'
+    %= javascript '/mojolicious/highlight.js/mojolicious.min.js'
+    %= javascript '/mojolicious/highlight.js/apache.min.js'
+    %= javascript '/mojolicious/highlight.js/ini.min.js'
+    %= javascript '/mojolicious/highlight.js/nginx.min.js'
+    %= javascript '/mojolicious/highlight.js/yaml.min.js'
```

Comment:
Why so many files? Why not bundle them into one file?

Reply:
Right, I was using the bundle in https://github.com/mojolicious/mojo/pull/1590 while developing, indeed its better to make a new bundle with these extra languages in.  Will do that later today. :muscle:

Reply:
Rebundled, thanks!